2018-11-04 22:20:07,364 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:59]
2018-11-04 22:20:21,184 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:59]
2018-11-04 22:23:45,659 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:59]
2018-11-04 22:23:49,879 ERROR: Exception on /reset_password_request [GET] [in C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\tmais\Documents\Coding\Python\hello_flask\app\routes.py", line 112, in reset_password_request
    title='Reset Password', form=form)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\templating.py", line 135, in render_template
    context, ctx.app)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\templating.py", line 117, in _render
    rv = template.render(context)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\tmais\Documents\Coding\Python\hello_flask\app\templates\reset_password_request.html", line 1, in top-level template code
    {% extends base.html %}
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\environment.py", line 430, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'base' is undefined
2018-11-04 22:25:40,876 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:59]
2018-11-04 22:28:54,087 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:59]
2018-11-08 20:13:19,162 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:61]
2018-11-08 20:20:52,762 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:61]
2018-11-08 20:33:16,070 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:63]
2018-11-08 20:36:13,757 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:63]
2018-11-10 18:08:09,682 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:65]
2018-11-10 18:08:40,322 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:65]
2018-11-10 18:11:29,721 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:65]
2018-11-10 18:13:19,035 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:65]
2018-11-10 18:14:23,813 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:65]
2018-11-10 18:15:41,612 INFO: Microblog startup [in C:\Users\tmais\Documents\Coding\Python\hello_flask\app\__init__.py:65]
2018-11-10 18:15:46,849 ERROR: Exception on /login [GET] [in C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\tmais\Documents\Coding\Python\hello_flask\app\routes.py", line 76, in login
    'login.html', title='Sign In', form=form)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\templating.py", line 135, in render_template
    context, ctx.app)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask\templating.py", line 117, in _render
    rv = template.render(context)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\tmais\Documents\Coding\Python\hello_flask\app\templates\login.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "C:\Users\tmais\Documents\Coding\Python\hello_flask\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 4, in block "doc"
    {%- block html %}
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block "html"
    {% block body -%}
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 21, in block "body"
    {% block navbar %}
  File "C:\Users\tmais\Documents\Coding\Python\hello_flask\app\templates\base.html", line 21, in block "navbar"
    <li><a href="{{ url_for('index') }}">{{ _('Home') }}</a></li>
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\ext.py", line 134, in _gettext_alias
    return __context.call(__context.resolve('gettext'), *args, **kwargs)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\jinja2\ext.py", line 140, in gettext
    rv = __context.call(func, __string)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask_babel\__init__.py", line 116, in <lambda>
    lambda x: get_translations().ugettext(x),
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask_babel\__init__.py", line 227, in get_translations
    [get_locale()],
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\flask_babel\__init__.py", line 261, in get_locale
    locale = Locale.parse(rv)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\babel\core.py", line 268, in parse
    parts = parse_locale(identifier, sep=sep)
  File "C:\Users\tmais\Envs\hello_flask\lib\site-packages\babel\core.py", line 1094, in parse_locale
    raise ValueError('expected only letters, got %r' % lang)
ValueError: expected only letters, got 'en-gb'
